app/src/main/java/pl/kossa/akainotes/fragments/viewNote/NoteViewViewModel.kt

```diff
@@ -0,0 +1,26 @@
+package pl.kossa.akainotes.fragments.viewNote
+
+import androidx.lifecycle.ViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+
+object NoteViewViewModel : ViewModel() {
```

Comment:
ViewModel powinen być klasą a nie object (ale to dotyczy też przekazywania argumentów)